internal/usage/usage.go

```diff
@@ -103,8 +103,8 @@ dbName and collection are only required for built-in roles.`
 	MeasurementType                           = "Measurements to return. If it is not specified, all measurements are returned."
 	FirstName                                 = "User's first name."
 	LastName                                  = "User's last name."
-	OrgRole                                   = "User's roles  for the associated organization."
-	ProjectRole                               = "User's roles  for the associated project."
+	OrgRole                                   = "User's roles  for the associated organization. Valid values are ORG_MEMBER, ORG_READ_ONLY, ORG_BILLING_ADMIN, ORG_GROUP_CREATOR, and ORG_OWNER."
+	ProjectRole                               = "User's roles  for the associated project. Valid Atlas CLI values are GROUP_OWNER, GROUP_READ_ONLY, GROUP_USER_ADMIN, GROUP_BILLING_ADMIN, GROUP_DATA_ACCESS_ADMIN, GROUP_DATA_ACCESS_READ_ONLY, GROUP_DATA_ACCESS_READ_WRITE, GROUP_CHARTS_ADMIN, and GROUP_CLUSTER_MANAGER. Valid MCLI values are all Atlas CLI values and GROUP_AUTOMATION_ADMIN, GROUP_BACKUP_ADMIN, GROUP_MONITORING_ADMIN."
```

Comment:
[q] instead of listing the values, could we share the API spec link where all the values are available? this ensures we'll not have an outdated doc as the list grows/changes and we would only update the API spec w/ openAPI 

[q2] can we create two variables one for mclI and another for atlas cli? that way we can default to the right variable based on the cli, I can help with that on the implementation e.g. `mcliOrgRole` `mcliProjectRole`

Reply:
We could link to the API spec for Atlas, but we don't have one for MMS and I don't see a place in the docs that the API roles are spelled out (just the UI ones), so those 3 additional roles would have to be listed. Would you like me to do that?

Reply:
Yes, maybe show atlas link for Atlas CLI and display the extra roles for mcli only. So `mcliOrgRole mcliProjectRole` and use it if `config.BinName ==  config.MongoCLI`